frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -340,7 +340,7 @@ const QueueItem: any = ({
           dateTested: shouldUseCurrentDateTime() ? null : dateTested,
         },
       })
-        .then(testResultsSubmitted)
+        .then(testResultsSubmitted, () => {})
```

Comment:
this is enough to make it so that a graphql error from test submission creates a toast (by getting caught below), instead of crashing the page. @nickscialli-usds is this okay?